frontend/src/app/services/audio.service.ts

```
@@ -0,0 +1,91 @@
+  unregister(name: string): void {
+    const entry = this.audios.get(name);
+    if (!entry) return;
+    try {
+      entry.element.pause();
+      entry.element.src = '';
+    } catch (e) {}
+    this.audios.delete(name);
+  }
+
+  unloadAll(): void {
+    this.audios.forEach((_, name) => this.unregister(name));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)